packages/sqflite/LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
```

Comment:
`sqflite` is licensed under BSD-2 and we usually follow the frontend package's license for our package in this repo. Are you sure to use the MIT license?